docs/source/index.rst

```diff
@@ -15,16 +14,22 @@ original system of `JupyterHub`_. As of now, you can find two main cases:
 1. `The Littlest JupyterHub <https://github.com/jupyterhub/the-littlest-jupyterhub>`__ distribution is suitable if you need a small number of users (1-100) and a single server with a simple environment.
 2. `Zero to JupyterHub with Kubernetes <https://github.com/jupyterhub/zero-to-jupyterhub-k8s>`__ allows you to deploy dynamic servers on the cloud if you need even more users.
 
+JupyterHub supports web browsers, tablets and smartphones and can be used by 
+multiple users at the same time such as; a class of students, a corporate data 
+science group or scientific research group. 
+JupyterHub makes life easier by enabling collaboration between both small (0-100 users) and 
```

Comment:
JupyterHub _doesn't_ really enable collaboration. It can be a tool in a collaborative environment, but I think this is overselling it.